doc/simulation/inspector/src/agent.py

```
@@ -5,6 +5,7 @@
 
 from cpu_usage import *
 from memory_usage import *
+from datetime import datetime, timedelta  # re-import after wildcard imports to avoid shadowing
```

Ruff (0.15.2)

[error] Redefinition of unused `datetime` from line 1: `datetime` redefined here

Remove definition: `datetime`

(F811)